server/myenv/Lib/site-packages/colorama/tests/initialise_test.py

```
@@ -0,0 +1,189 @@
+                sys.stdout = stdout
+
+                stderr = Mock()
+                stderr.closed = False
+                stderr.isatty.return_value = True
+                stderr.fileno.return_value = 2
+                sys.stderr = stderr
+
+            for native_ansi in [False, True]:
+                with patch(
+                    'colorama.ansitowin32.enable_vt_processing',
+                    lambda *_: native_ansi
```

Ruff (0.12.2)

Function definition does not bind loop variable `native_ansi`

(B023)